webpack.config.js

```diff
@@ -401,7 +401,12 @@ if (!IS_PRODUCTION) {
 if (FORCE_WEBPACK_DEV_SERVER || (USE_HOT_MODULE_RELOAD && !NO_DEV_SERVER)) {
   const backendAddress = `http://localhost:${SENTRY_BACKEND_PORT}/`;
 
-  appConfig.plugins.push(new webpack.HotModuleReplacementPlugin());
+  // Hot reload react components on save
+  // We include the library here as to not break docker/google cloud builds
```

Comment:
I think this won't be necessary if you use `IS_DEV_ENV` above as suggested?

Reply:
It is because this plugin is a devDep

Reply:
Yes so it should not be used when in prod mode?

Reply:
Right I guess it’s more referring to the require statement needing to be there

Reply:
I'm asking whether we can wrap these two lines in an if statement:

```js
if (SHOULD_HOT_MODULE_RELOAD) {
  const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
  appConfig.plugins.push(new ReactRefreshWebpackPlugin());
}
```

Reply:
Ah, moved around some logic